tests/lit-tests/targets_x86_llvm14plus.ispc

```diff
@@ -0,0 +1,13 @@
+// The test checks that target definitions (including all synonyms) are successfully consumed by compiler.
+
+//; RUN: %{ispc} %s -o %t.o --nostdlib --target=avx512skx-x4
```

Comment:
Is the intention really check *skx-x* targets or *spr-x* targets?

Reply:
Good catch, fixed.